corehq/ex-submodules/pillowtop/pillow/interface.py

```diff
@@ -429,6 +436,8 @@ def __init__(self, name, checkpoint, change_feed, processor,
             self.processors = [processor]
 
         self._change_processed_event_handler = change_processed_event_handler
+        self.is_dedicated_migration_process = is_dedicated_migration_process
+        self.process_num = process_num
```

Comment:
Where is `process_num` used? If it's just in the place where I commented above then I think you can remove it entirely